src/Reflection/ClassReflection.php

```diff
@@ -933,7 +936,7 @@ public function getResolvedPhpDoc(): ?ResolvedPhpDocBlock
 			return null;
 		}
 
-		$docComment = $this->reflection->getDocComment();
+		$docComment = $this->reflectionDocComment ?? $this->reflectionDocComment = $this->reflection->getDocComment();
```

Comment:
I'd like to use a control structure (an `if`) here instead of just `??` on a single line. Like this:

```php
if ($this->reflectionDocComment !== null) {
    $docComment = $this->reflectionDocComment;
} else {
    $docComment = $this->reflection->getDocComment();
    $this->reflectionDocComment = $docComment;
}
```